UltiSnips/javascript.snippets

```diff
@@ -68,7 +68,7 @@ ${1:var }${2:function_name} = function $2`!p snip.rv = space_before_function_par
 }`!p snip.rv = semi(snip)`
 endsnippet
 
-snippet af "Anonymous Function" i
+snippet anf "Anonymous Function" i
```

Comment:
Why have we changed this?

Reply:
it conflicted with `af` for arrow functions.